projects/storefrontlib/src/lib/cart/cart.module.ts

```diff
@@ -1,28 +1,13 @@
 import { NgModule } from '@angular/core';
 
-import { EffectsModule } from '@ngrx/effects';
-import { StoreModule } from '@ngrx/store';
-
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
-
-import { services } from './facade/index';
-import { guards } from './guards/index';
 import { AddToCartModule } from './components/add-to-cart/add-to-cart.module';
 import { CartDetailsModule } from './components/cart-details/cart-details.module';
 import { CartSharedModule } from './components/cart-shared/cart-shared.module';
-import { effects } from './store/effects/index';
-import { reducerToken, reducerProvider } from './store/reducers/index';
-import { metaReducers } from './store/reducers/index';
-
+import { CartNotEmptyGuard } from './guards/cart-not-empty.guard';
 @NgModule({
-  imports: [
-    AddToCartModule,
-    CartDetailsModule,
-    CartSharedModule,
-    NgbModule,
-    StoreModule.forFeature('cart', reducerToken, { metaReducers }),
-    EffectsModule.forFeature(effects)
-  ],
-  providers: [reducerProvider, ...services, ...guards]
+  imports: [AddToCartModule, CartDetailsModule, CartSharedModule, NgbModule],
+  exports: [AddToCartModule, CartDetailsModule, CartSharedModule],
+  providers: [CartNotEmptyGuard]
 })
 export class CartModule {}
```

Comment:
We now have 2 equally named modules. I would prefer to rename this one: i.e. `CartComponentModule`.